src/logo/builtin.c

```diff
@@ -522,6 +522,50 @@ static const FFlogo* getLogoCentOSSmall()
     FF_LOGO_RETURN
 }
 
+static const FFlogo* getLogoCrystalLinux()
+{
+    FF_LOGO_INIT
+    FF_LOGO_NAMES("crystal", "Crystal", "crystal-linux", "Crystal-Linux")
+    FF_LOGO_LINES(
+"                       mysssym\n"
+"                     mysssym\n"
+"                   mysssym\n"
+"                 mysssym\n"
+"               mysssyd\n"
+"             mysssyd    N\n"
+"           mysssyd    mysym\n"
+"         mysssyd      dysssym\n"
+"       mysssyd          dysssym\n"
+"     mysssyd              dysssym\n"
+"     mysssyd              dysssym\n"
+"       mysssyd          dysssym\n"
+"         mysssyd      dysssym\n"
+"           mysym    dysssym\n"
+"             N    dysssym\n"
+"                dysssym\n"
+"              dysssym\n"
+"            dysssym\n"
+"          dysssym\n"
+<<<<<<< HEAD
+"        dysssym\n"
+
+
+    )              
+=======
+"        dysssym\n"            
+>>>>>>> 5f8a27faa53c1beba2131d4a1638119ac478dff9
```

Comment:
Please remove this part from the pull request, this is not a patch

Reply:
How 😅

Reply:
Remove it from you local code. Then:
```bash
git add .
git commit -m "Removed patch code"
git push
```

Reply:
Ohh that sorry i don't know how it got there :(

Reply:
let me fix it